server/game/cards/07-WotW/PlazaOfPride.js

```diff
@@ -1,35 +1,27 @@
 const DrawCard = require('../../drawcard.js');
 
 class PlazaOfPride extends DrawCard {
-
     setupCardAbilities(ability) {
         this.action({
             title: 'Stand character',
             cost: [
                 ability.costs.kneelSelf(),
                 ability.costs.discardFromHand()
             ],
+            target: {
+                cardCondition: (card, context) =>
+                    card.location === 'play area'
+                    && card.getType() === 'character'
+                    && card.kneeled
+                    && card.getCost() <= context.costs.discardFromHand.getCost() + 3
```

Comment:
Because targets are checked before costs are paid, this will likely crash (probably a reason this card wasn't converted to the target API earlier). You can look at Tower of the Hand for an example to get around this.